test-framework/junit5-internal/src/main/java/io/quarkus/test/ConfigUtil.java

```diff
@@ -0,0 +1,30 @@
+package io.quarkus.test;
+
+import io.quarkus.runtime.configuration.QuarkusConfigFactory;
+
+public final class ConfigUtil {
```

Comment:
Not another `ConfigUtil`... we already have many of these (or similar). I know this will be for test only, and this method shouldn't go the main one, but I already get frustrated when I search for it to get 3 or 4 alternatives and always pick the wrong one :)

Maybe this could go into `io.quarkus.test.junit.launcher.ConfigUtil`?

Reply:
Unfortunately I don't have access to `io.quarkus.test.junit.launcher.ConfigUtil`  from `quarkus-junit5-internal`, because it's in its sibling module `quarkus-junit5`...

I'll merge the two `ConfigUtils` and move the result to `quarkus-test-common`, with a more unique name (`TestConfigUtil`). If you think `io.quarkus.test.junit.launcher.ConfigUtil` is API, I can leave it in place and deprecate it; just let me know.

Reply:
I don't think it is an API class. Feel free to merge and move :)